events-api/src/modules/tickets/tickets.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { validateRequestBody } from "#/shared/validators/request-body.validator";
+import { Router } from "express";
+import { ticketsCreateDtoSchema } from "./dto/requests/tickets-create-dto";
+import { TicketsService } from "./tickets.service";
+import { validateRouteParams } from "#/shared/validators/route-params.validator";
+import { eventsRouteParamsDtoSchema } from "../events/dto/requests/events-route-params.dto";
+
+export const TicketsController = Router();
+
+TicketsController.post(
+  "/",
+  validateRequestBody(ticketsCreateDtoSchema),
+  async (req, res) => {
+    await TicketsService.createTicket(req.body);
```

Comment:
What does happen if the event is not found?

Reply:
![image](https://github.com/user-attachments/assets/6dfd2bae-9899-41ef-b0c1-c5a8a6b64697)